hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/audit/impl/LoggingAuditor.java

```diff
@@ -251,11 +251,10 @@ private void attachRangeFromRequest(AmazonWebServiceRequest request) {
           return;
         }
         if (rangeValue.length != 2) {
-          WARN_INCORRECT_RANGE.warn("Expected range to contain 0 or 2 elements. Got "
-              + rangeValue.length + " elements. Ignoring");
+          WARN_INCORRECT_RANGE.warn("Expected range to contain 0 or 2 elements. Got {} elements. Ignoring.", rangeValue.length);
```

Comment:
ok, checkstyle is unhappy. needs splitting.

i will do that myself locally as i do a retest and merge.

Reply:
Sorry, I always miss that one. I wish it would fail the Yetus approval